integration/simple-cache-client.test.ts

```diff
@@ -152,42 +223,97 @@ describe('SimpleCacheClient.ts Integration Tests', () => {
       const cacheKey = v4();
       // Create a longer cache value that should take longer than 1ms to send
       const cacheValue = new TextEncoder().encode(v4().repeat(1000));
-      await expect(
-        shortTimeoutClient.set(cacheName, cacheKey, cacheValue)
-      ).rejects.toThrow(TimeoutError);
+      const setResponse = await shortTimeoutClient.set(
+        cacheName,
+        cacheKey,
+        cacheValue
+      );
+      expect(setResponse instanceof CacheSet.Error).toEqual(true);
+      if (setResponse instanceof CacheSet.Error) {
+        expect(setResponse.errorCode()).toEqual(MomentoErrorCode.TIMEOUT_ERROR);
+      }
     });
   });
   it('should set and then delete a value in cache', async () => {
     const cacheKey = v4();
     const cacheValue = new TextEncoder().encode(v4());
     const momento = new SimpleCacheClient(AUTH_TOKEN, 1111);
     await momento.set(INTEGRATION_TEST_CACHE_NAME, cacheKey, cacheValue);
-    const getSuccess = await momento.get(INTEGRATION_TEST_CACHE_NAME, cacheKey);
-    // validate set worked correctly
-    expect(getSuccess.bytes()).toEqual(cacheValue);
+    const getResponse = await momento.get(
+      INTEGRATION_TEST_CACHE_NAME,
+      cacheKey
+    );
+    expect(getResponse instanceof CacheGet.Error).toEqual(false);
+    expect(getResponse instanceof CacheGet.Miss).toEqual(false);
+    if (getResponse instanceof CacheGet.Hit) {
+      expect(getResponse.valueBytes()).toEqual(cacheValue);
+    }
 
-    await momento.delete(INTEGRATION_TEST_CACHE_NAME, cacheKey);
+    const deleteResponse = await momento.delete(
+      INTEGRATION_TEST_CACHE_NAME,
+      cacheKey
+    );
+    expect(deleteResponse instanceof CacheDelete.Success).toEqual(true);
     const getMiss = await momento.get(INTEGRATION_TEST_CACHE_NAME, cacheKey);
-    expect(getMiss.status).toEqual(CacheGetStatus.Miss);
+    expect(getMiss instanceof CacheGet.Miss).toEqual(true);
+  });
+  it('should return a miss response for a nonexistent cache key', async () => {
+    const cacheKey = v4();
+    const momento = new SimpleCacheClient(AUTH_TOKEN, 1111);
+    const deleteResponse = await momento.delete(
+      INTEGRATION_TEST_CACHE_NAME,
+      cacheKey
+    );
+    expect(deleteResponse instanceof CacheDelete.Error).toEqual(false);
+    const missResponse = await momento.get(
+      INTEGRATION_TEST_CACHE_NAME,
+      cacheKey
+    );
+    expect(missResponse instanceof CacheGet.Miss).toEqual(true);
```

Comment:
This seems redundant with the previous test.

Reply:
Good eye, I deleted it.